documentation/modules/metrics/proc_metrics-grafana-dashboard.adoc

```diff
@@ -7,22 +7,20 @@
 = Enabling the example Grafana dashboards
 
 [role="_abstract"]
-Use Grafana dashboards to visualize Strimzi data.
-Add dashboards if you are using Prometheus and Grafana to extract metrics and monitor Strimzi.
-
-Strimzi provides xref:ref-metrics-config-files-{context}[example dashboard configuration files for Grafana].
-Example dashboards are provided in the `examples/metrics/grafana-dashboards` directory as JSON files:
-
-* `strimzi-kafka.json`
-* `strimzi-zookeeper.json`
-* `strimzi-operators.json`
-* `strimzi-kafka-connect.json`
-* `strimzi-kafka-mirror-maker-2.json`
-* `strimzi-kafka-bridge.json`
-* `strimzi-cruise-control.json`
-* `strimzi-kafka-exporter.json`
-
-The example dashboards are a good starting point for monitoring key metrics, but they do not represent all available metrics.
+Use Grafana to provide visualizations of Prometheus metrics.
```

Comment:
_Suggested change:_

Use Grafana to provide visualizations of Prometheus metrics on customizable dashboards.